x-pack/metricbeat/module/aws/utils.go

```diff
@@ -188,8 +188,8 @@ func GetResourcesTags(svc resourcegroupstaggingapiiface.ClientAPI, resourceTypeF
 		}
 
 		getResourcesInput.PaginationToken = output.PaginationToken
-		if resourceTypeFilters == nil || len(output.ResourceTagMappingList) == 0 {
-			return nil, nil
+		if len(output.ResourceTagMappingList) == 0 {
```

Comment:
Thank you for fixing this! I almost created a PR to fix pagination in aws module yesterday but you beat me to it!! One question, what do you think about using the paginator in aws sdk? For example here: https://github.com/aws/aws-sdk-go-v2/blob/main/service/resourcegroupstaggingapi/api_op_GetResources.go#L261

Reply:
yes，SDK paginator is easy and safe.

Reply:
It is done.